game/kernel/asm_funcs_x86_64.asm

```diff
@@ -10,12 +6,11 @@ SECTION .text
 global _arg_call_systemv
 _arg_call_systemv:
   pop rax
-  push r10
-  push r11
-  sub rsp, 8
+  push r10 ; arg 6 (arg in GOAL only so must be preserved)
+  push r11 ; arg 7 (arg in GOAL only so must be preserved)
```

Comment:
Minor nit - but the reason to preserve this isn't because it's an argument in GOAL only, it's because the OpenGOAL compiler expects this register to be saved but systemv doesn't save it.